backend/devices/bitbox02/keystore_simulator_test.go

```diff
@@ -465,7 +508,11 @@ func TestSimulatorSignBTCTransactionSendSelfSameAccount(t *testing.T) {
 
 		// Before simulator v9.20, address confirmation data was not written to stdout.
 		if device.Version().AtLeast(semver.NewSemVer(9, 20, 0)) {
-			require.Contains(t, stdOut.String(), "ADDRESS: This BitBox02: bc1pg848p0rvmj0r3j064prlpw0gecyzkwlpt7ndzdlmh2mvkyu299psetgxhf")
+			if device.Version().AtLeast(semver.NewSemVer(9, 22, 0)) {
```

Comment:
No need for nesting here, >=v9.22 implies >=v9.20. 

Better imho: `if >= v9.22 { ...} else if >= v9.20 {... }`

Reply:
Thanks, done